### PR TITLE
Allow to specify clientProperties

### PR DESCRIPTION
**Pull request by ptusch**
clientProperties will give us hints about whatever we want. We could specify it in the amqpProtocolAdapter or as client option.

### CODE REVIEW

File: src/amqp/connection.js [hunk 1]
```diff
@@ -101,6 +107,7 @@ var Adapter = function( parameters ) {
 	var passphrase = getOption( parameters, "RABBIT_PASSPHRASE" ) || getOption( parameters, "passphrase" );
 	var pfxPath = getOption( parameters, "RABBIT_PFX" ) || getOption( parameters, "pfxPath" );
 	var useSSL = certPath || keyPath || passphrase || caPaths || pfxPath;
+  var clientProperties = getOption( parameters, "clientProperties");
```
Comment (stheuer) [inline src/amqp/connection.js]: intendation

File: src/amqp/connection.js [hunk 2]
```diff
@@ -126,12 +133,8 @@ var Adapter = function( parameters ) {
 	if ( useSSL ) {
 		this.protocol = 'amqps://';
 	}
-	this.options.clientProperties = {
-		host: info.host(),
-		process: info.process(),
-		lib: info.lib()
-	};
 	this.limit = max( this.servers.length, this.ports.length );
+  this.options.clientProperties = Object.assign(CLIENT_PROPERTIES, clientProperties);
```
Comment (stheuer) [inline src/amqp/connection.js]: intendation

**Review by stheuer — CHANGES_REQUESTED** [review]
Please Fix two intendations and merge

File: src/amqp/connection.js (hunk 1 above)
Comment (ptusch) [reply src/amqp/connection.js]: Done: 1eca555

File: src/amqp/connection.js (hunk 2 above)
Comment (ptusch) [reply src/amqp/connection.js]: Done: 1eca555

**Review by stheuer — APPROVED** [review]

**Pull request merged**